blablahelp-backend/src/main/java/org/nnn4eu/hfische/blablahelp/blablahelpbackend/config/LocalRunner.java

```diff
@@ -0,0 +1,80 @@
+package org.nnn4eu.hfische.blablahelp.blablahelpbackend.config;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.nnn4eu.hfische.blablahelp.blablahelpbackend.account.Account;
+import org.nnn4eu.hfische.blablahelp.blablahelpbackend.account.AccountService;
+import org.nnn4eu.hfische.blablahelp.blablahelpbackend.account.ERole;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+@Profile("dev | local")
+public class LocalRunner implements ApplicationRunner {
```

Comment:
Warum hast du diese Klasse geschrieben? Um nur mal die Datenbankverbindung zu testen find ich das ein bissche viel und sollte auch wenns nicht in der Productiv umgebeung läuft nicht mit auf main